Arch Linux linyaps-box-git 编译错误

```bash
==> Retrieving sources...
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
-> Creating working copy of linyaps-box-git git repo...
Cloning into 'linyaps-box-git'...
done.
==> Starting prepare()...
==> Starting pkgver()...
==> Starting build()...
-- The CXX compiler identification is GNU 15.2.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- make dev build
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.5.1")
-- Checking for module 'libcap>=2.25'
--   Found libcap, version 2.76
-- CPM: Using local package nlohmann_json@3.12.0
-- CPM: Adding package CLI11@2.4.1 (v2.5.0)
-- Looking for C++ include linux/openat2.h
-- Looking for C++ include linux/openat2.h - found
-- Found GTest: /usr/lib/cmake/GTest/GTestConfig.cmake (found version "1.17.0")
-- Configuring done (3.3s)
-- Generating done (0.0s)
-- Build files have been written to: /build/linyaps-box-git/src/linyaps-box-git/build
ninja: Entering directory `build'
[1/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/cgroup_manager.cpp.o
[2/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/app.cpp.o
[3/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/impl/disabled_cgroup_manager.cpp.o
[4/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/command/run.cpp.o
[5/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/command/kill.cpp.o
[6/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/container.cpp.o
FAILED: CMakeFiles/linyaps-box.dir/src/linyaps_box/container.cpp.o
/usr/bin/c++ -DLINYAPS_BOX_ACTIVE_LOG_LEVEL=LOG_WARNING -DLINYAPS_BOX_CLONE_CHILD_STACK_SIZE="(1U << 20)" -DLINYAPS_BOX_DEFAULT_LOG_LEVEL=LOG_WARNING -DLINYAPS_BOX_ENABLE_CAP -DLINYAPS_BOX_HAVE_OPENAT2_H -DLINYAPS_BOX_STACK_GROWTH_DOWN=true -I/build/linyaps-box-git/src/linyaps-box-git/src -I/build/linyaps-box-git/src/linyaps-box-git/build/src -isystem /build/linyaps-box-git/src/linyaps-box-git/build/_deps/cli11-src/include -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection         -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -Wp,-D_GLIBCXX_ASSERTIONS -g -ffile-prefix-map=/build/linyaps-box-git/src=/usr/src/debug/linyaps-box-git -flto=auto -std=c++17 -fmacro-prefix-map=/build/linyaps-box-git/src/linyaps-box-git=. -MD -MT CMakeFiles/linyaps-box.dir/src/linyaps_box/container.cpp.o -MF CMakeFiles/linyaps-box.dir/src/linyaps_box/container.cpp.o.d -o CMakeFiles/linyaps-box.dir/src/linyaps_box/container.cpp.o -c /build/linyaps-box-git/src/linyaps-box-git/src/linyaps_box/container.cpp
In file included from /usr/include/c++/15.2.1/cassert:46,
from /build/linyaps-box-git/src/linyaps-box-git/src/linyaps_box/container.cpp:29:
/build/linyaps-box-git/src/linyaps-box-git/src/linyaps_box/container.cpp: In function ‘int {anonymous}::container_ns::clone_fn(void*)’:
/build/linyaps-box-git/src/linyaps-box-git/src/linyaps_box/container.cpp:1658:17: error: ‘struct {anonymous}::clone_fn_args’ has no member named ‘self_socket’
1658 |     assert(args.self_socket.get() >= 0);
|                 ^~~~~~~~~~~
[7/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/container_ref.cpp.o
[8/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/command/list.cpp.o
[9/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/command/exec.cpp.o
[10/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/container_status.cpp.o
[11/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/impl/table_printer.cpp.o
[12/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/impl/json_printer.cpp.o
[13/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/config.cpp.o
[14/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/impl/status_directory.cpp.o
[15/39] Building CXX object CMakeFiles/linyaps-box.dir/src/linyaps_box/command/options.cpp.o
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in build().
Aborting...
```

--- a/src/linyaps_box/container.cpp
+++ b/src/linyaps_box/container.cpp
@@ -1655,7 +1655,7 @@ try {
 
     auto &args = *static_cast<clone_fn_args *>(data);
 
-    assert(args.self_socket.get() >= 0);
+    assert(args.socket.get() >= 0);
     std::set<uint> except_fds{
         STDIN_FILENO,
         STDOUT_FILENO,

